aict_tools/cta_helpers.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+from astropy.coordinates import EarthLocation
+from astropy.coordinates import AltAz
+from astropy.time import Time
+import logging
+import warnings
+
+from astropy.coordinates import SkyCoord, AltAz
+from ctapipe.coordinates import (
```

Comment:
Only CameraFrame should be needed here.

Reply:
Jup, that just grew expoentially over time